docs/parcels.md

```diff
@@ -0,0 +1,149 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be mounted manually 
+by an application. Parcels use similar methodology as applications but are mounted by a manual function call rather than the activity function.
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Typically we recommend you mount a parcel within 
+the context of an application because the parcel will be unmounted with the application.
+
+## Parcel configuration
+A parcel is just an object with 3 or 4 functions on it, it can be loaded syncronously or asyncronously.
+Each function is considered a lifecycle method, `bootstrap`, `mount`, `unmout`, `update` and must return a promise.
+When creating a parcel it's strongly recommended that you use the [lifecycle helper methods](/docs/single-spa-ecosystem.md#help-for-frameworks).
+An example of a parcel written in React would look like this:
+```js
+// myParcel.js
+import React from 'react'
+import ReactDom from 'react-dom'
+import singleSpaReact from 'single-spa-react'
+import MyParcelComponent from './my-parcel-component.component.js'
+export const MyParcel = singleSpaReact({
+  React,
+  ReactDom,
+  rootComponent: MyParcelComponent
+})
+
+// in this case singleSpaReact is taking our inputs and generating an object with the required lifecycles.
+```
+Then to use the parcel you just created all you need to do is use the `Parcel` component provided in [single-spa-react](https://github.com/CanopyTax/single-spa-react#parcels)
+```js
+// mycomponent.js
+import { Parcel } from 'single-spa-react'
+import MyParcel from './myparcel.js'
+
+export myComponent extends React.Component {
+    render () {
+      return (
+        <Parcel
+          config={MyParcel}
+          { /* optional props */ }
+          { /* and any extra props you want here */ }
+        />
+      )
+    }
+}
+```
+Note in some cases the optional props are required [additional examples](https://github.com/CanopyTax/single-spa-react#examples)
+
+## Parcel Lifecycles
+Start with [applications](/docs/applications.md#registered-application-lifecycle) to learn more about the functionality of single-spa's lifecycle methods.
+
+### Bootstrap
+This lifecycle function will be called once, right before the parcel is
+mounted for the first time.
+
+```js
+function bootstrap(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you do one-time initialization
```

Comment:
indentation here is weird -- the lines above this one use two spaces but this line uses four spaces